php/class-coauthors-guest-authors.php

```diff
@@ -187,11 +187,11 @@ function handle_create_guest_author_action() {
 		}
 
 		if ( ! wp_verify_nonce( $_GET['nonce'], 'create-guest-author' ) ) {
-			wp_die( __( "Doin' something fishy, huh?", 'co-authors-plus' ) );
+			wp_die( esc_html_e( "Doin' something fishy, huh?", 'co-authors-plus' ) );
```

Comment:
All `wp_die` calls with a translation should use `esc_html__` (instead of ``esc_html_e`)